permissions_validators/src/public_blockchain/mod.rs

```diff
@@ -698,4 +720,11 @@ mod tests {
             .check(&bob_id, &set, &wsv)
             .is_deny());
     }
+
+    #[test]
+    fn default_permission_tokens_are_unique() {
+        let mut permissions = default_permission_token_ids();
+        permissions.sort();
+        assert!(permissions.windows(2).all(|wnd| wnd[0] != wnd[1]));
+    }
```

Comment:
```suggestion
    #[test]
    fn default_permission_tokens_are_unique() {
        let permissions_set = HashSet::from(default_permission_token_ids());
        assert_eq!(permissions_set.len(), default_permission_token_ids().len());
    }
```

This should work faster and looks better, IMO.

Your approach has `O(NlogN) + 2*O(N)` complexity and mine has `O(N) * Amortized_O(1)`